py_zipkin/_encoding_helpers.py

```diff
@@ -1,12 +1,163 @@
 import socket
 from collections import namedtuple
 
+from enum import Enum
+
+from py_zipkin.exception import ZipkinError
+
+
 Endpoint = namedtuple(
     'Endpoint',
     ['service_name', 'ipv4', 'ipv6', 'port'],
 )
 
 
+_V1Span = namedtuple(
```

Comment:
I wonder if it would make more sense to have this subclass some common span base class?

Reply:
Created `SpanBuilder` class